README.md

```diff
@@ -2,50 +2,47 @@
 
 ![CSI Logo](logo.png)
 
-This project contains the CSI [specification](spec.md). Please see the
-[build reference](#build-reference) section for information on how to
-generate the protobuf or Go source files.
+This project contains the CSI [specification](spec.md).
+Please see the [build reference](#build-reference) section for information on how to generate the protobuf and build for each language.
 
-## Build Reference
-This section describes how to generate the CSI protobuf file, generate
-Go sources from the protobuf, and which environment variables can
-influence the build process.
+## Build Reference.
+
+[CMake](https://cmake.org/cmake) is required for building.
 
 ### Generate Protobuf
-The CSI protobuf can be generated with the following command:
 
 ```bash
-$ make csi.proto
+$ mkdir build
+$ cd build
+$ cmake ..
+$ make proto
 ```
 
-In the above example the protobuf file is generated by parsing a `spec.md`
-from a remote git repository (this one in fact). However, it's also
-possible to generate the protobuf from a local spec file:
+In the above example, the protobuf file is generated by parsing `spec.md`. `csi.proto` will be generated in the build directory.
+
+### Generate Go Source
+
+The following command generates the Go source file from the protobuf:
 
 ```bash
-$ CSI_SPEC_FILE=spec.md make csi.proto
+$ mkdir build
```

Comment:
if the point of including golang build support here is to encourage people to reuse these bits then ... we've failed. this is too much to ask of golang devs. a typical golang devs workflow is `go get github.com/org/project/package`.

this all feels too complex for what the original CI scripts were attempting to do: validate that the protos in the spec actually compiled to something. I somewhat regret accepting a PR that complicated the process. This PR adds additional prerequisites and process .. things that I don't think are actually appropriate for this repo.

I guess I could be convinced otherwise, but as of this moment I don't see the justification for making this repo more complex.